### PR TITLE
Fix qpid queue creation

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: containers/roles/foreman/files/create_qpid_queue.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-qpid-config -b tcp://${QPID_SERVICE}:${QPID_PORT} queues katello_event_queue
+qpid-config -b ${QPID_PORT} queues katello_event_queue
 
 if [ ?! != 0 ]; then
-  qpid-config -b tcp://${QPID_SERVICE}:${QPID_PORT} add queue katello_event_queue --durable
+  qpid-config -b ${QPID_PORT} add queue katello_event_queue --durable
```
Comment (carbonin) [inline containers/roles/foreman/files/create_qpid_queue.sh:6]: Is this supposed to be the built-in variable? I think it's `QPID_SERVICE_PORT` / `QPID_SERVICE_HOST`

Comment (carbonin) [reply containers/roles/foreman/files/create_qpid_queue.sh:6]: Just deployed and this is what I have in the foreman container for QPID:

```bash
sh-4.2# env | sort | grep -i qpid
QPID_PORT=tcp://172.30.219.102:5672
QPID_PORT_5672_TCP=tcp://172.30.219.102:5672
QPID_PORT_5672_TCP_ADDR=172.30.219.102
QPID_PORT_5672_TCP_PORT=5672
QPID_PORT_5672_TCP_PROTO=tcp
QPID_SERVICE_HOST=172.30.219.102
QPID_SERVICE_PORT=5672
QPID_SERVICE_PORT_PORT_5672_TCP=5672
```

Comment (carbonin) [reply containers/roles/foreman/files/create_qpid_queue.sh:6]: Ah you're actually using the `QPID_PORT` one, lgtm then :smile:

Comment (ehelms) [reply containers/roles/foreman/files/create_qpid_queue.sh:6]: I was also surprised that QPID_PORT had IP and PORT but it seems to be a consistent pattern.

**Pull request merged**